os/fs/procfs/fs_procfsproc.c

```diff
@@ -474,8 +474,6 @@ static ssize_t proc_cmdline(FAR struct proc_file_s *procfile, FAR struct tcb_s *
 		copysize = procfs_memcpy(procfile->line, linesize, buffer, remaining, &offset);
 
 		totalsize += copysize;
-		buffer += copysize;
-		remaining -= copysize;
```

Comment:
It looks dead but this is necessary.
checking total size is missed after this.
So we need to add it before return. instead removing this